test/functional/apps/visualize/_linked_saved_searches.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import expect from 'expect.js';
+
+export default function ({ getPageObjects, getService }) {
+  const retry = getService('retry');
+  const PageObjects = getPageObjects(['common', 'discover', 'visualize', 'header']);
+
+  describe('visualize app', function describeIndexTests() {
```

Comment:
I would like us to not have 2 levels of describe in these tests unless it's breaking down different features.  The parent index.js already has a describe `visualize app` so this one is redundant and make it difficult to debug when a failure happens in the before or after methods.
Could you just remove this outer one and move the before and after methods into the `describe('linked saved search')` one?